app/report/residency_fr_taxes_fr/ReportResidencyFrTaxesFr.tsx

```diff
@@ -293,6 +330,44 @@ export const ReportResidencyFrTaxesFr: React.FunctionComponent<
               <Page510 taxes={taxes} isPrintMode={isPrintMode} />
             </div>
           </Section>
+          <Section title="Source of information">
+            <div>Some external sources are used to compute data:</div>
+            <ul className="list-disc pl-6 mt-2">
+              <li>
+                <a
```

Comment:
Can you use [<Link>](https://github.com/hinosxz/tax-helper/blob/main/app/guide/shared/ui/Link.tsx) instead of a for external links? Feel free to update the styles if you don't like them but using a single component will help keeping things consistent